administration/src/components/generation/CardCreationModel.ts

```diff
@@ -0,0 +1,22 @@
+import {CardType} from "../../models/CardType";
+
+const MAX_NAME_LENGTH = 100
+
+export interface CardCreationModel {
+    id: number,
+    forename: string,
+    surname: string,
+    expirationDate: Date,
+    cardType: CardType
+}
+
+export const isForenameValid = (value: string) => value.length > 0 && value.length < MAX_NAME_LENGTH
+
+export const isSurnameValid = (value: string) => value.length > 0 && value.length < MAX_NAME_LENGTH
```

Comment:
Just a trifle, but I would rather not have to identical functions for that as long as we do not handle forename and surname differently (which has a high chance of never happening)